### ISSUE TITLE
Could not get Accessibility View to work with hovers

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#25970]
Testing -186214

1. Install GitHub Pull Request extension. Open GitHub view on the side. Hover over an issue, and put focus in it
2. Ctrl + F2 to open Accessibility View. Nothing happens 🐛 

I could not get the Accessibility View to work with any hover. What am I missing here?
